core/templates/components/rubrics-editor/rubrics-editor.component.html

```diff
@@ -65,20 +65,32 @@
             </button>
             <button type="button"
                     class="btn btn-success protractor-test-save-rubric-explanation-button"
-                    [disabled]="!this.isExplanationValid(this.rubric.getDifficulty(), idx)"
+                    [disabled]="!this.isExplanationValid(this.rubric.getDifficulty(), idx)
+                      || !this.isExplanationLengthValid(this.rubric.getDifficulty(), idx)"
                     (click)="this.saveExplanation(this.rubric.getDifficulty(), idx)">
               Save
             </button>
           </div>
+          <br>
+          <div *ngIf="!this.isExplanationLengthValid(this.rubric.getDifficulty(), idx)" class="rubric-error-message">
+            Explanations should have at most 300 characters.
+          </div>
+          <div class="oppia-card-content-limit-error" *ngIf="this.isTotalExplanationsLengthReached()">
+            There can be maximum of 10 explanations across all levels.
```

Comment:
"length" in these two function names seem to refer to different things, which seems a bit confusing. Could we call the 2nd one "hasReachedExplanationCountLimit" or similar?

Reply:
Sure